cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -268,12 +270,7 @@ parameter_types! {
 		[Parachain(ASSET_HUB_ID), PalletInstance(ASSETS_PALLET_ID), GeneralIndex(RESERVABLE_ASSET_ID.into())]
 	);
 
-	/// The Penpal runtime is utilized for testing with various environment setups.
-	/// This storage item provides the opportunity to customize testing scenarios
-	/// by configuring the trusted asset from the `SystemAssetHub`.
-	///
-	/// By default, it is configured as a `SystemAssetHubLocation` and can be modified using `System::set_storage`.
-	pub storage CustomizableAssetFromSystemAssetHub: Location = SystemAssetHubLocation::get();
```

Comment:
cc @bkontur 😅

Reply:
hmm, with `CustomizableAssetFromSystemAssetHub` you could do the same:
```
PenpalA::execute_with(|| {
		assert_ok!(<PenpalA as Chain>::System::set_storage(
			<PenpalA as Chain>::RuntimeOrigin::root(),
			vec![(
				CustomizableAssetFromSystemAssetHub::key().to_vec(),
				Location::new(2, []).encode(),
			)],
		));
	});
```

The point of `CustomizableAssetFromSystemAssetHub` feature is that any emulator-tests/runtime can customize it for their tests. It is kind of generic, but `pub ForeignAssetPrefix: Location = (Parent, Parent).into();` is hard-coded so nobody will be able to customize. 
I am not sure, but I think we use penpal not only for eth-bridge testing, there are other tests e.g. relay -> AH -> penpal and so on. Also maybe Acala or other parachain can write their own xcm-emulator tests with penpal, so they won't use `(Parent, Parent)`...

The penpal should be generic testing runtime with some features, so I would revert this and rather remove all mentions of bridgehub, e.g.:
```
- /// Another pallet assets instance to store foreign assets from bridgehub.
+ /// Another pallet assets instance to store foreign assets.  
```

Reply:
Can we make `CustomizableAssetFromSystemAssetHub`  default to `(Parent, Parent).into()`? Which I believe is a sane default for most cases. While being "hard-coded", is actually a very open filter, and will accept any asset from a different consensus system. So it will work for your bridge as well. It will allow any foreign asset from Asset Hub to be used with the Foreign Assets pallet instance on Penpal.

Reply:
> ... default to (Parent, Parent).into() ...
> It will allow any foreign asset from Asset Hub 

This is not exactly true, e.g. on AH you can have foreignAssets from different local parachain, e.g. `Location(1, Parachain(2059))`, so (Parent, Parent) won't work for them.

I think we can go with default like you said `CustomizableAssetFromSystemAssetHub = (Parent, Parent).into()`.

But what is the real problem here? Why is `Penpal::set_storage(CustomizableAssetFromSystemAssetHub::key(), ...)` problem?


The penpal should not even care about "bridge" word.